edx_rest_framework_extensions/middleware.py

```diff
@@ -158,3 +159,83 @@ def _set_request_auth_type_metric(self, request):
         else:
             auth_type = 'session-or-unknown'
         monitoring.set_custom_metric('request_auth_type', auth_type)
+
+
+class JwtAuthCookieMiddleware(object):
```

Comment:
Nit: This file seems to becoming a heterogenous collection of various middlewares.  At some point we'll want to refactor edx-drf-extensions and move authn or jwt related code into its own sub-folder.

Reply:
In progress here: https://github.com/edx/edx-drf-extensions/pull/45